ja.yml

```diff
@@ -134,182 +134,181 @@ app:
+      creditsLink: "https://github.com/reindex-ot"
     updates:
-      title: "アップデートを確認"
-      description: "定期的に新しいバージョンのWindhawkとインストールされているmodを確認する。"
+      title: "更新を確認"
+      description: "Windhawk とインストールされている Mod の新しいバージョンを定期的に確認します。"
     devMode:
       title: "開発者モード"
-      description: "modの作成・修正など、開発者向けのオプションを表示します。"
-    advancedSettings: "詳細設定"
+      description: "Mod の作成や変更などの開発者向けのアクションを表示します。"
+    advancedSettings: "高度な設定"
     hideTrayIcon:
       title: "トレイアイコンを隠す"
-      description: "Windhawkを終了するには、このオプションを無効にする必要があります。"
+      description: "Windhawk を終了するには、このオプションを無効化する必要があります。"
     requireElevation:
-      title: "Windhawkの実行にUACの昇格を必要とする"
-      description: "シングルユーザーのコンピュータでのUACプロンプトの煩わしさを回避します。このオプションを有効にすると、Windhawkの実行にUACの昇格が必要になります。"
+      title: "Windhawk の実行時に UAC の昇格を要求"
+      description: "Windhawk の実行には管理者権限が必要ですが、シングルユーザーコンピューターの環境で毎回 UAC のプロンプトが表示されるのは煩わしいため、Windhawk はこれを回避しています。Windhawk の実行に UAC の昇格を要求させるには、このオプションを有効化してください。"
     dontAutoShowToolkit:
-      title: "ツールキットダイアログを自動表示しない"
-      description: "デフォルトの状態では、Windhawkがシステムの不安定化やその他の理由でタスクバーにアクセスできないことを検出すると、自動的にツールキットダイアログを表示します。ツールキットダイアログでは、Windhawkの終了、セーフモードへの切り替え、その他の操作を行うことができます。ツールキットダイアログは、Ctrl+Win+Wのキーボードショートカットでも表示することができます。"
+      title: "ツールキットダイアログを自動で表示しない"
+      description: "既定では、システムが不安定な状態やその他の理由でタスクバーにアクセスできない状態を検出すると、Windhawk は自動的にツールキットダイアログを表示します。ツールキットダイアログは、Windhawk の終了、セーフモードの切り替え、その他の操作を行なうことができます。ツールキットダイアログは、Ctrl+Win+W のキーボードショートカットでも表示できます。"
     modInitDialogDelay:
-      title: "Mod初期化ダイアログの遅延"
-      description: "modの初期化時にプログレスダイアログを表示するまでの待ち時間(ミリ秒)を指定します。"
+      title: "Mod 初期化ダイアログの遅延"
+      description: "Mod の初期化でプログレスダイアログを表示するまでの待機時間 (ミリ秒) を設定できます。"
     moreAdvancedSettings:
-      title: "さらに詳細設定を表示"
-      restartNotice: "設定を適用するためにWindhawkを再起動します。"
-      saveButton: "保存してWindhawkを再起動"
+      title: "その他の高度な設定"
+      restartNotice: "Windhawk を再起動すると設定が適用されます。"
+      saveButton: "保存をして Windhawk を再起動"
       cancelButton: "キャンセル"
     loggingVerbosity:
-      appLoggingTitle: "Windhawkログの冗長性"
-      engineLoggingTitle: "Windhawkエンジンのログの冗長性"
-      description: "ログはDebugViewなどのツールで見ることができます"
+      appLoggingTitle: "Windhawk ログの冗長性"
+      engineLoggingTitle: "Windhawk エンジンのログの冗長性"
+      description: "ログはデバッグビューなどのツールで確認することができます。"
       none: "なし"
       error: "エラー"
       verbose: "冗長"
     processList:
       titleExclusion: "プロセス除外リスト"
-      descriptionExclusion: "Windhawkがインジェクトしないプロセス名/パスのリストです。Windhawkが特定のプログラムと互換性がない場合に便利です。このリストにプロセスを追加すると、Windhawkがそれをカスタマイズするのを防ぐだけでなく、このプロセスが作成する子プロセスをWindhawkが傍受するのを防ぐことに注意してください。このため、このような場合、Windhawkはmodの読み込みに若干の遅れを生じます。"
-      exclusionCriticalProcessesNotice: "The special <0>{{critical_system_processes}}</0> value can be added to the list to exclude critical system processes. Beware of removing it, as crashing one of these processes might lead to a BSOD."
-      titleInclusion: "プロセス包含リスト"
```

Comment:
I also noticed that this line was removed. I'll restore it, let me know if a better translation can be used.

Reply:
I haven't deleted anything, I've just made corrections and added translations.

```
titleExclusion: "プロセス除外リスト"
descriptionExclusion: "Windhawk がインジェクションをしないプロセスとパスのリストです。Windhawk が特定のプログラムと互換性がない場合に役立ちます。このリストにプロセスを追加すると、Windhawk がそのプロセスをカスタマイズしなくなるだけでなくそのプロセスが作成する、子プロセスを Windhawk が傍受できなくなることに注意してください。場合によっては Windhawk の Mod の読み込みに若干の遅延が発生します。"
exclusionCriticalProcessesNotice: "<0>{{critical_system_processes}}</0> という特別な値をリストに追加することで、重要なシステムプロセスを除外することができます。これらのプロセスをクラッシュさせた場合、BSOD が発生する可能性があるので、削除しないように注意してください。"
titleInclusion: "プロセス包含リスト"
```

Reply:
You can compare the files, `titleInclusion` is gone in the updated file.

Before your change, line 244:
https://github.com/ramensoftware/windhawk-translate/blob/74a08ad9b0a97a2819ce2b90beb1f318e8cb7800/ja.yml#L244

After your change:
https://github.com/ramensoftware/windhawk-translate/blob/d61ba98002f28d93025fae4056cb889f4f5687ac/ja.yml#L244

That's OK, I restored this line.